server/http_handler.go

```diff
@@ -773,6 +775,13 @@ func NewFrameItemFromRegionKey(key []byte) (frame *FrameItem, err error) {
 	frame = &FrameItem{}
 	frame.TableID, frame.IndexID, frame.IsRecord, err = tablecodec.DecodeKeyHead(key)
 	if err == nil {
+		if frame.IsRecord {
+			_, frame.RecordID, err = tablecodec.DecodeRecordKey(key)
+		} else {
+			_, _, frame.IndexValues, err = tablecodec.DecodeIndexKey(key)
```

Comment:
Could we get recordID from the no-unique index?

Reply:
It's hard to know the type of the index here. I think the best we can do it to show the recordID in index values (the last item).

Reply:
Did we write `IndexValues` into the response?